src/Goerli-DssSpell.sol

```diff
@@ -19,10 +19,10 @@ pragma solidity 0.6.12;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-interface DssVestLike {
-    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
-    function restrict(uint256) external;
-}
+// interface DssVestLike {
+//     function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+//     function restrict(uint256) external;
+// }
```

Comment:
```suggestion
```

Reply:
I need this in the mainnet spell next, so I was just leaving it there.